libs/gltfio/src/MorphHelper.cpp

```diff
@@ -162,6 +164,18 @@ void MorphHelper::addPrimitive(cgltf_mesh const* mesh, int primitiveIndex, Entit
                 }
             }
         }
+
+        // Generate flat normals if necessary.
+        if (!hasNormals && !prim.material->unlit) {
+            TangentsJob job;
+            TangentsJob::Params params = { .in = { &prim, targetIndex } };
+            TangentsJob::run(&params);
+            if (params.out.results) {
+                morphHelperPrim.targets->setTangentsAt(engine, targetIndex,
+                        params.out.results, params.out.vertexCount);
+                free(params.out.results);
```

Comment:
It would be nicer to have `TangentsJob::free(&params)` to not have this "unbalanced" call to free.